apps/apollo-stories/src/components/ContentItemDuoAction/ContentItemDuoAction.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from "@storybook/react";
+import bank from "@material-symbols/svg-700/rounded/account_balance_wallet-fill.svg";
+import { ContentItemDuoAction } from "@axa-fr/design-system-apollo-react/lf";
+import { ComponentProps } from "react";
+
+const meta: Meta<typeof ContentItemDuoAction> = {
+  component: ContentItemDuoAction,
+  title: "Components/ContentItemDuoAction",
```

Comment:
Tes args se répètent dans les stories.
Tu as la possibilité d'en définir par défaut directement dans la constante meta comme ceci : 

```suggestion
  title: "Components/ContentItemDuoAction",
  args: {
    contentItemProps: {
      type: "icon",
      icon: bank,
      title: "Texte principale",
      subtitle1: "Texte secondaire",
      subtitle2: "Texte tertiare",
    },
  },
```
Comme ça on définit la valeur par défaut de `contentItemProps` sans pour autant en définir pour `mode` qui change de valeur en fonction de la story.